### PR TITLE
BZ#1890607 - Add OpenShift SDN egress IP sharing limitation

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1890607

Preview: [Egress IP address assignment for project egress traffic](https://deploy-preview-28666--osdocs.netlify.app/openshift-enterprise/latest/networking/openshift_sdn/assigning-egress-ips.html#nw-egress-ips-about_egress-ips)

### CODE REVIEW

**Comment by jboxman** [comment]
@danwinship,

This adds a missing limitation for the OpenShift SDN egress IPs feature. Can you confirm whether this update makes sense?

Thanks!

File: modules/nw-egress-ips-about.adoc [hunk 1]
```diff
@@ -148,8 +148,15 @@ When the unreachable node comes back online, the egress IP address automatically
...
-You cannot use manually assigned and automatically assigned egress IP addresses on the same nodes.
-If you manually assign egress IP addresses from an IP address range, you must not make that range available for automatic IP assignment.
+The following limitations apply when using egress IP addresses with the OpenShift SDN cluster network provider:
+
+- You cannot use manually assigned and automatically assigned egress IP addresses on the same nodes.
+- If you manually assign egress IP addresses from an IP address range, you must not make that range available for automatic IP assignment.
+- You cannot share egress IP addresses across multiple namespaces using the OpenShift SDN egress IP address implementation.
+ifeval::["{product-version}" >= "4.6"]
++
+If you need to share IP addresses across namespaces, the OVN-Kubernetes cluster network provider egress IP address implementation allows you to span IP addresses across multiple namespaces.
```
Comment (jeana-redhat) [inline modules/nw-egress-ips-about.adoc:158]: There's nothing wrong with this, but the way it's formatted makes it look like there is a missing bullet in the output. Would it work to not force it onto the next line?

**Review by jeana-redhat — APPROVED** [review]
LGTM

File: modules/nw-egress-ips-about.adoc (hunk 1 above)
Comment (jboxman) [reply modules/nw-egress-ips-about.adoc:158]: @jeana-redhat, wasn't sure if that would work, but it does!

**Pull request merged**